frontend/packages/console-shared/src/utils/resource-utils.ts

```diff
@@ -692,7 +672,6 @@ export class TransformResourceData {
       };
       const status = resourceStatus(obj, current, isRollingOut);
       const pods = [..._.get(current, 'pods', []), ..._.get(previous, 'pods', [])];
-      const events = this.getEventsForPods(pods);
```

Comment:
I think `pods` above was introduced in https://github.com/openshift/console/pull/3961/  to fetch events for that pod , as we are fetching events now based on selector so we can remove this as well

Reply:
we use pods in the objects in other places (donut and such). I don't believe they were introduced in that PR on changed to create the `pods` constant rather than inline.

Reply:
Yes , got it.